saleor/product/utils/product_variants.py

```diff
@@ -0,0 +1,21 @@
+from ...graphql.core.utils import from_global_id_strict_type
+from ...product.models import Product
+
+
+def get_products_without_variants(products_list):
+    """Return list of product's ids without variants."""
+    products_ids = get_products_from_global_ids(products_list)
```

Comment:
If this function is a global utility function (not living in the GraphQL-layer) I think it should already receive resolved product objects and not GraphQL IDs that you have to resolve here.